templates/resources/arm/aks.json

```diff
@@ -0,0 +1,80 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"clusterName": {
+			"type": "string"
+		},
+		"dnsPrefix": {
+			"type": "string"
+		},
+		"clusterLocation": {
+			"type": "string"
+		},
+		"servicePrincipalId": {
+			"type": "securestring"
+		},
+		"servicePrincipalKey": {
+			"type": "securestring"
+		},
+		"agentCount": {
+			"defaultValue": 3,

```

Comment:
Should we set it to a lower count - 1 or 2?

Reply:
I've set it to 1 for now